qubesdbus/models.py

```diff
@@ -90,6 +90,15 @@ def Start(self):
         vm.start()
         return True
 
+    @dbus.service.method("org.qubes.Domain", in_signature="s", 
+        out_signature="b")
+    def RunService(self, service):
+        app = qubesadmin.Qubes()
+        name = str(self.name)
+        vm = app.domains[name]
+        vm.run_service(service)
```

Comment:
`vm.run_service()` returns `subprocess.Popen` instance, which can be used to communicate with the service. If you don't want that (which is ok for this API), better add `stdin=subprocess.DEVNULL, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL`, otherwise some service may wait indefinitely on input/buffer space for output.

Reply:
Good catch. I've updated the PR.